package.json

```diff
@@ -47,10 +47,10 @@
     "react-router-dom": "^4.2.2",
     "react-select": "^1.1.0",
     "style-loader": "0.19.0",
-    "sw-precache-webpack-plugin": "0.11.4",
+    "sw-precache-webpack-plugin": "^0.11.5",
     "sweetalert2": "^7.0.8",
     "url-loader": "0.6.2",
-    "web3": "^1.0.0-beta.26",
+    "web3": "^1.0.0-beta.34",
```

Comment:
@varasev I would prefer to fixate the versions of packages to prevent problems, when the new minor version of package will break DApp functionality. What do you think?

Reply:
@vbaranov yes, I've fixated them in https://github.com/poanetwork/poa-dapps-voting/pull/143/commits/826fb47fd96994d98ddb8a02a18271f0ae7e799e.